### PR TITLE
Update app

### PR DESCRIPTION
**Pull request by SavvyShah**
Update the [airbnb-clone-app](https://medium.appbase.io/how-to-build-an-airbnb-clone-with-react-and-elasticsearch-8eb91e7e85e2)
Following things are updated:
- Newer versions of `reactivesearch`, `reactivemaps`, `react`
- Uses `Searchbox` instead of `DataSearch`

[Link to app](https://codesandbox.io/s/github/SavvyShah/Airbeds-app/tree/update-app)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,10 +6,12 @@
     "prettier": "^1.18.2"
   },
   "dependencies": {
-    "@appbaseio/reactivemaps": "^3.0.0-beta.4",
-    "@appbaseio/reactivesearch": "^3.3.4",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
+    "@appbaseio/react-searchbox": "^1.7.0",
```
Comment (mohdashraf010897) [inline package.json]: @SavvyShah I don't think we need "@appbaseio/react-searchbox" for this example.

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
-  RangeSlider
+  RangeInput,
+  SelectedFilters
 } from "@appbaseio/reactivesearch";
+
+import './App.css'
 import { ReactiveGoogleMap } from "@appbaseio/reactivemaps";
-import "./App.css";
 
 class App extends Component {
```
Comment (mohdashraf010897) [inline src/App.js]: @SavvyShah I would suggest you use a functional component instead of a class component.

File: src/App.js [hunk 3]
```diff
@@ -32,8 +58,9 @@ class App extends Component {
     return (
       <div className="main-container">
         <ReactiveBase
-          app="airbeds-test-app"
-          credentials="X8RsOu0Lp:9b4fe1a4-58c6-4089-a042-505d86d9da30"
+          app="clone-airbeds"
+          url="https://73afb5484d0e:26bd5cb0-1afc-4e19-8870-4a2eda8d0b56@appbase-demo-ansible-abxiydt-arc.searchbase.io"
+          enableAppbase
           type="listing"
```
Comment (mohdashraf010897) [inline src/App.js]: @SavvyShah remove `type` prop.

File: src/App.js [hunk 4]
```diff
@@ -1,15 +1,41 @@
...
+  SelectedFilters
 } from "@appbaseio/reactivesearch";
+
+import './App.css'
 import { ReactiveGoogleMap } from "@appbaseio/reactivemaps";
-import "./App.css";
 
 class App extends Component {
+
+	dateQuery(value) {
```
Comment (mohdashraf010897) [inline src/App.js]: @SavvyShah Can you try not using this customQuery? I doubt it is required now.

File: src/App.js [hunk 5]
```diff
@@ -119,36 +148,41 @@ class App extends Component {
...
               showMarkerClusters={false}
-              showSearchAsMove={false}
+              showSearchAsMove
               innerClass={{
                 label: "label"
               }}
               renderAllData={(
                 hits,
-                streamHits,
                 loadMore,
```
Comment (mohdashraf010897) [inline src/App.js]: @SavvyShah Not needed

File: src/App.js [hunk 6]
```diff
@@ -119,36 +148,41 @@ class App extends Component {
...
                 renderMap,
-                renderPagination
+                renderPagination,
+                triggerClickAnalytics,
```
Comment (mohdashraf010897) [inline src/App.js]: not needed

File: src/index.js [hunk 7]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
```
Comment (mohdashraf010897) [inline src/index.js]: Can we have a single index.js file? i don't think there's any need for multiple files as such. :)

**Review by mohdashraf010897 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 6 above)
Comment (SavvyShah) [reply src/App.js]: Sorry, I didn't get it.

File: src/index.js (hunk 7 above)
Comment (SavvyShah) [reply src/index.js]: Fixed

File: src/App.js (hunk 3 above)
Comment (SavvyShah) [reply src/App.js]: Fixed

File: src/App.js (hunk 2 above)
Comment (SavvyShah) [reply src/App.js]: Fixed

File: src/App.js (hunk 4 above)
Comment (SavvyShah) [reply src/App.js]: No it doesn't work. I tried using it without customQuery.

Comment (SavvyShah) [reply src/App.js]: Tried different values for `queryFormat`. Tried using `basic_date` and `date`.

File: src/index.js [hunk 8]
```diff
@@ -1,8 +1,237 @@
...
+              }}
+              className="rightCol"
+              showMarkerClusters={false}
+              showSearchAsMove
+              innerClass={{
+                label: "label"
+              }}
+              renderAllData={(
+                hits,
+                loadMore,
```
Comment (mohdashraf010897) [inline src/index.js:184]: @SavvyShah loadMore and triggerClickAnalytics are not used, so remove them. :)

**Review by mohdashraf010897 — APPROVED** [review]

**Pull request merged**